matlab_simulation/05-estimation/ekf/ekf_example.m

```diff
@@ -0,0 +1,98 @@
+% Extended Kalman filter example
+%
+% Radar measurement of airplane position while flying at a constant
+% altitude and velocity. In this case the motion model is linear, but the
+% measurement model is nonlinear.
+%
+% This example can be found on slide 111 of Estimation-I in the notes.
+clear;clc;
+
+%% Create AVI object
+makemovie = 1;
+if(makemovie)
+    vidObj = VideoWriter('ekf.avi');
+    vidObj.Quality = 100;
+    vidObj.FrameRate = 8;
+    open(vidObj);
+end
+
+% Discrete time step
+dt = 0.1;
+
+% Initial State
+x0 = [20 -2 3]';
```

Comment:
Agree with this suggestion. You may add comments at the top of "elf_example.m", like what you did in "ekf.m".